src/pathogen_cluster/__init__.py

```diff
@@ -0,0 +1,35 @@
+"""
+pathogen-cluster.
+
+"""
+
+__version__ = "1.0.0"
+__author__ = 'Sravani Nanduri, John Huddleston'
+__credits__ = 'Bedford Lab, Vaccine and Infectious Disease Division, Fred Hutchinson Cancer Research Center, Seattle, WA, USA'
+
+import argparse
+import sys
+from .pathogen_cluster import pathogen_cluster
+
+def make_parser():
+    parser = argparse.ArgumentParser(description = "Reduced dimension embeddings for pathogen sequences", formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    # correct

```

Comment:
What does this comment refer to? Is it a TODO?

Reply:
I should definitely delete this - a relic of me updating the command line arguments 😅